test/TestOrchestrations.cs

```diff
@@ -12,20 +12,36 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
     internal static class TestOrchestrations
     {
         public const char BigValueChar = '*';
+        public static bool SayHelloWithActivityForRewindShouldFail = true;
 
         public static string SayHelloInline([OrchestrationTrigger] DurableOrchestrationContext ctx)
         {
             string input = ctx.GetInput<string>();
             return $"Hello, {input}!";
         }
 
+
         public static async Task<string> SayHelloWithActivity([OrchestrationTrigger] DurableOrchestrationContext ctx)
         {
             string input = ctx.GetInput<string>();
             string output = await ctx.CallActivityAsync<string>(nameof(TestActivities.Hello), input);
             return output;
         }
 
+
+        public static async Task<string> SayHelloWithActivityForRewind([OrchestrationTrigger] DurableOrchestrationContext ctx)
+        {
+            string input = ctx.GetInput<string>();
+            string output = await ctx.CallActivityAsync<string>(nameof(TestActivities.Hello), input);

```

Comment:
We will want to validate that the activity didn't get called a second time. Probably the simplest thing we can do is check the trace output (currently commented-out in the new test you added).